web_poet/testing/utils.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+from pathlib import Path
+from typing import Any, Iterable, Union
+
+from itemadapter import ItemAdapter
+
+from web_poet.serialization import SerializedDataFileStorage, serialize
+
+INPUT_DIR_NAME = "inputs"
+OUTPUT_FILE_NAME = "output.json"
```

Comment:
I think I might be missing it but what's the current proposed structure inside this file?

Reply:
It's a `json.dumps` of the to_item() result.

Reply:
A `json.dumps` of `ItemAdapter(page_object.to_item()).asdict()`, to be exact. Line 40 below.

I wonder if the web-poet standard should require the output of `to_item()` to be an itemadapter-supported item. We could make that a test-only requirement, but for the sake of standardization I would go for a general requirement.

Reply:
+1, I think it makes sense to document it. 

It is not a limitation on the item type returned, because one can register custom adapters.